db/migrations/postgres/000063_create_identities_table.up.sql

```diff
@@ -0,0 +1,137 @@
+BEGIN;
+
+CREATE TABLE identities (
+  seq                   SERIAL          PRIMARY KEY,
+  id                    UUID            NOT NULL,
+  did                   VARCHAR(256)    NOT NULL,
+  parent                UUID,
+  messages_claim        UUID            NOT NULL,
+  messages_verification UUID,
+  messages_update       UUID,
+  itype                 VARCHAR(64)     NOT NULL,
+  namespace             VARCHAR(64)     NOT NULL,
+  name                  VARCHAR(64)     NOT NULL,
+  description           VARCHAR(4096)   NOT NULL,
+  profile               TEXT,
+  created               BIGINT          NOT NULL,
+  updated               BIGINT          NOT NULL
+);
+
+CREATE UNIQUE INDEX identities_id ON identities(id);
+CREATE UNIQUE INDEX identities_did ON identities(did);
+CREATE UNIQUE INDEX identities_name ON identities(itype, namespace, name);
+
+CREATE TABLE verifiers (
+  seq            SERIAL          PRIMARY KEY,
+  id             UUID            NOT NULL,
+  identity       UUID            NOT NULL,
```

Comment:
What is the difference between "id" and "identity" here? My first guess was that "id" is a generated ID unique to the verifier row, and that "identity" is a foreign key reference to the identities table. But I saw in the migrations logic below, these columns are set to the same value.

Reply:
`identity` is a link to the `Identity` resource that owns this.
`id` is the UUID of this `Verification` object.

Reply:
OK, that's what I'd guessed after a few minutes staring. Why are they set to the same value on the migration path, instead of generating a unique `id`? Is that purely to support the down-migration strategy of separating migrated from newly-created entries?

Reply:
1. to support the down migration strategy
2. to avoid needing to have functions in PSQL/SQLite to generate UUIDs

Reply:
I realized actually after doing the migration work, that the `id` of a verifier is really a `localId` (not an `id`) as it won't be applied universally to all nodes. I'm trying to think in the context of a DID document if this is a problem.... it might be.

Reply:
Yea, if we can deterministically compute something like a `hash`, that seems better. Generally we only use `id` when the ID can be explicitly shared in a definition message.

Reply:
I'm going to have to do a post migration during startup for this though sadly :(

Reply:
actually I might cheat and put the identity ID in there for migrations, and we can document that in the release notes

Reply:
This feels good to me. Suggested a few comments here in the migration file for anyone looking, but this definitely needs to make it into release notes as well.